README.md

```diff
@@ -9,16 +9,17 @@ may well be used to fire requests to other services.
 
 
 * System dependencies
-    * none
+    * postgres 10.5  -> see setup below
 
 * Configuration
+    you may have to install pstgres gem if you dont already have it 
+    
+    ```gem install pg```
```

Comment:
So both this and `bundle install` fail for me with this error:

```
$ gem install pg -- --with-pg-config /usr/local/opt/libpq/bin/pg_config
Building native extensions with: '--with-pg-config /usr/local/opt/libpq/bin/pg_config'
This could take a while...
ERROR:  Error installing pg:
	ERROR: Failed to build gem native extension.

    current directory: /Users/grahambleach/.gem/ruby/2.5.1/gems/pg-1.0.0/ext
/Users/grahambleach/.rubies/ruby-2.5.1/bin/ruby -r ./siteconf20180816-32064-1la9lpe.rb extconf.rb --with-pg-config /usr/local/opt/libpq/bin/pg_config
Using config values from true
checking for libpq-fe.h... no
Can't find the 'libpq-fe.h header
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
...
```

Reply:
So the commands that are currently missing are:

```
brew install libpq                                          # needed for the pg gem's native extensions
PATH=$PATH:/usr/local/opt/libpq/bin        # so that the pg gem can find the pq library
```

After this, `bundle install` works fine.

Reply:
I didn't have any problems with `gem install pg`, which is what's in the readme. I get the same error as you when I run `gem install pg -- --with-pg-config /usr/local/opt/libpq/bin/pg_config` though. Why do we neeed the `--with-pg-config` flag?

Reply:
We don't. The `--with-pg-config` option was an unsuccessful attempt to fix the failure to build native extension.

Reply:
Ah, I see. Ignore my subsequent comments in that case.